apiV2/controller/propertyController.js

```diff
@@ -1,41 +1,47 @@
 import Property from '../models/property';
 import { propertys } from '../data/data';
-import { Agent,User } from '../models/users';
+import { Agent, User } from '../models/users';
+import Resp from '../helpers/response';
 
 class PropertyController {
   postProperty(req, res) {
-      const id = propertys.length + 1;
-      const { price, address, city, state, type, imageUrl } = req.body;
-      const { password,isAdmin,token, ...noA } = res.locals.user;
-      const property = new Property(id, noA, price, address, city, state, type, imageUrl); 
-      Agent.createProperty(property);
-      console.log(process.env.NODE_ENV)
-      return res.status(201).send({ status:201, property });
-    };
+    const {
+      price, address, city, state, type, imageUrl,
+    } = req.body;
+    const { id } = res.locals.user;
+    const adObj = new Property(id, price, address, city, state, type, imageUrl);
+    const property = adObj.addProperty(adObj);
+    property.then((e) => {
+      console.log(req.path);
+      return Resp(201, 'Property created', e.rows[0], res);
```

Comment:
please give this a more descriptive name.. resp is not descriptive.. also you do not start a function name with an upper case letter (airbnb guide). please make adjustment